pepper/exo_compute/private_input_trade_execution.cpp

```diff
@@ -0,0 +1,102 @@
+#include <iostream>
```

Comment:
I would suggest using another file name for this script. 

something like "optimizer_input_conversion"